lib/core/routing/app_navigation.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:caramelseed/core/routing/routes.dart';
+import 'package:caramelseed/core/widgets/onBoarding/onboarding.dart';
+import 'package:caramelseed/features/authentication/presentation/pages/login_page.dart';
+import 'package:caramelseed/features/item_list/presentation/pages/item_list_page.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:page_transition/page_transition.dart';
+
+import '../../main.dart';
+
+class AppNavigation {
+  PageTransition<dynamic> getTransition(RouteSettings settings, Widget child) {
```

Comment:
Para el proximo PR podemos usar esto 
https://medium.com/flutter-community/page-transitions-using-themedata-in-flutter-c24afadb0b5d
y evitamos la lib transition.